profiles/license_groups

```diff
@@ -72,7 +72,7 @@ BINARY-REDISTRIBUTABLE @FREE bh-luxi Broadcom Dina intel-ucode ipw2100-fw ipw220
 
 # License agreements that try to take away your rights. These are more
 # restrictive than "all-rights-reserved" or require explicit approval.
-EULA Adaptec-EULA AdobeFlash-11.x ArxFatalis-EULA-JoWooD baudline BCS bf1942-lnxded CAPYBARA-EULA Coherent-Graphics CROSSOVER CROSSOVER-2 CYANA DOOM3 ETQW FAH-EULA-2009 FAH-EULA-2014 FraunhoferFDK GameFront Gameplay-Group-EULA GIMPS GOG-EULA google-chrome googleearth Google-TOS Intel-SDP Introversion LastPass LOKI-EULA LRCTF MakeMKV-EULA MARBLEBLAST Mendeley-terms MSjdbcEULA40 MTA-0.5 NVIDIA-CODEC-SDK NVIDIA-CUDA Nero-AAC-EULA Nero-EULA-US OPERA-12 OPERA-2014 Oracle-BCLA-JavaSE PAPERS-PLEASE POMPOM postal2 Primate-Plunge PUEL Q3AEULA Q3AEULA-20000111 QUAKE4 Quartus-prime-megacore RAR RTCW RTCW-ETEULA RuneScape-EULA SJ-Labs skype-4.0.0.7-copyright spin-commercial spin-educational StarUML-EULA TeamViewer THINKTANKS UPEK-SDK-EULA ut2003 ut2003-demo Vivaldi
+EULA Adaptec-EULA AdobeFlash-11.x ArxFatalis-EULA-JoWooD baudline BCS bf1942-lnxded CAPYBARA-EULA Coherent-Graphics CROSSOVER CROSSOVER-2 CYANA DOOM3 ETQW FAH-EULA-2009 FAH-EULA-2014 FraunhoferFDK GameFront Gameplay-Group-EULA GIMPS GOG-EULA google-chrome googleearth Google-TOS Intel-SDP Introversion LastPass LOKI-EULA LRCTF MakeMKV-EULA MARBLEBLAST Mendeley-terms MSjdbcEULA40 MTA-0.5 NVIDIA-CODEC-SDK NVIDIA-CUDA Nero-AAC-EULA Nero-EULA-US OPERA-12 OPERA-2014 Oracle-BCLA-JavaSE PAPERS-PLEASE POMPOM postal2 Primate-Plunge PUEL Q3AEULA Q3AEULA-20000111 QUAKE4 Quartus-prime-megacore RAR RTCW RTCW-ETEULA RuneScape-EULA SJ-Labs skype-4.0.0.7-copyright spin-commercial spin-educational StarUML-EULA TeamViewer THINKTANKS UPEK-SDK-EULA ut2003 ut2003-demo Vivaldi worklog-assistant
```

Comment:
*shrug* Another of these licenses whose author believes that they would be enforceable. Hint: If you can acquire a copy legally (and you can, since apart from the no-cost restriction the program can be distributed freely) then in many legislations (including the EU and the USA) you are allowed to use it, without any need to agree to their legal bullshit.

But yes, if we take it serious then it should be added to @EULA as well.

Disclaimer: IANAL, TINLA.